x-pack/plugins/cases/tsconfig.json

```diff
@@ -69,6 +69,7 @@
     "@kbn/core-theme-browser",
     "@kbn/serverless",
     "@kbn/core-http-server",
+    "@kbn/alerting-plugin",
```

Comment:
It was added automatically 🤷‍♀️

Reply:
we should remove it, I do not see any needs for it

Reply:
It was added back by CI after removing it https://github.com/elastic/kibana/pull/163037/commits/6e0a44ca43c9e67b4f9e7669fc2bd3febc9a2526